packages/talker/lib/src/talker.dart

```diff
@@ -96,7 +96,10 @@ class Talker implements TalkerInterface {
 
   /// {@macro talker_history}
   @override
-  List<TalkerDataInterface> get history => _history;
+  List<TalkerDataInterface> get history =>
+      (_logger as TalkerLogger).settings.displayLatestFirst
+          ? _history.reversed.toList()
+          : _history;
```

Comment:
It is better not to change the logic here too

Reply:
Please move this logic to TalkerScreenController

Reply:
Refactored